libraries/AP_Scripting/docs/docs.lua

```diff
@@ -4212,3 +4212,45 @@ function crsf:get_menu_event(events) end
 ---@param data string -- binary encoded response payload
 ---@return boolean -- true if the repsonse was successfully sent, false otherwise
 function crsf:send_write_response(data) end
+
+-- handle for DroneCAN message operations
+---@class DroneCAN_Handle_ud
+local DroneCAN_Handle_ud = {}
+
+-- create a DroneCAN_Handle, needed for all other DroneCAN message operations
+---@param driver_index number -- DroneCAN driver index, 0 for first driver
+---@param signature uint64_t_ud -- message signature
+---@param data_type number -- message data type ID
+---@return DroneCAN_Handle_ud
+function DroneCAN_Handle(driver_index, signature, data_type) end
+
+-- create a DroneCAN_Handle, needed for all other DroneCAN message operations
+---@param driver_index number -- DroneCAN driver index, 0 for first driver
+---@param signature uint64_t_ud -- message signature
+---@param data_type number -- message data type ID
+---@param canfd boolean -- send as CANFD
```

Comment:
I think the only difference here is this argument, you can mark it as optional with a question mark.

```suggestion
---@param canfd? boolean -- send as CANFD
```

That should allow you to remove the other copy.

Reply:
thanks, changed